packages/sdk/src/EventSenderEngine.ts

```diff
@@ -80,9 +80,9 @@ export class EventSenderEngine {
 
   send(context: Value.Struct, name: string, message?: Value.Struct): void {
     this.writeQueue.push({
-      eventDefinition: `eventDefinitions/${name}`,
```

Comment:
Not a deal breaker, but in other platform we have the Client adding this prefix. Especially when storage is used, it's nice to save events without this backend-specific requirement, and have the Client code handle that